service/src/main/java/org/jboss/sbomer/rest/v1alpha1/SBOMResource.java

```diff
@@ -140,11 +142,11 @@ public Sbom getById(@PathParam("id") String id) {
     }
 
     @GET
-    @Path("/build/{id}")
+    @Path("/build/{buildId}")
     @Operation(
             summary = "Get all SBOMs related to a PNC build",
             description = "Get all the SBOMs related to the specified PNC build, paginated.")
```

Comment:
I actually wonder if we should paginate it. Thoughts? Pagination is great if we are talking about UI, but for REST endpoints it may be unexpected. Thoughts?

Reply:
Maybe for this one we could avoid the pagination as there should not be a lot of SBOM associated with a build. But I would keep it for the general listAll endpoint.